DB/dbclient.py

```diff
@@ -0,0 +1,20 @@
+import sqlite3
+
+
+class DB:
+    def __init__(self, path):
+        self.db = sqlite3.connect(path)
+        self.cursor = self.db.cursor()
+
+    def add_user(self, login, password):
```

Comment:
Class DB should have method connected to work with database: connect, select, disconnect

Reply:
Its enough to have staticmethods: connect, select, insert, disconnect; without __init__ method for class DB?

Reply:
Class DB should work with database. It should not know anything about logins, users and etc.